Flask_app/models/models.py

```diff
@@ -0,0 +1,43 @@
+from sqlalchemy import Column , Integer , String 
+from Flask_app import db
+from sqlalchemy.ext.hybrid import hybrid_property
+from werkzeug.security import(
+    generate_password_hash,
+    check_password_hash
+    )
+
+
+class User(db.Model):
+
+    __tablename__ = "Users"
+
+
+    id= Column(Integer,primary_key=True)
+    email=Column(String(300),nullable=False)
+    username=Column(String(300),nullable=False)
+    firstname=Column(String(20),nullable=False)
+    lastname=Column(String(20),nullable=False)
+    mobile_number=Column(String(20),nullable=False)
+    hash_password = Column('password', String(350), nullable=False)
+    age = db.Column(db.Integer)
```

Comment:
Remove this age column and add a dob column, User will enter their date of birth and using property, The age will be calculated.